src/linter.ts

```diff
@@ -411,6 +447,28 @@ export default class ShellCheckProvider implements vscode.CodeActionProvider {
     await vscode.window.showTextDocument(doc, { preview: true });
   }
 
+  private async disableCheckForLine(
+    textDocument: vscode.TextDocument,
+    ruleId: string,
+    range: vscode.Range,
+  ) {
+    if (!this.isAllowedTextDocument(textDocument)) {
+      return;
+    }
+    let position = new vscode.Position(
+      range.start.line,
+      textDocument.lineAt(range.start.line).range.start.character,
+    );
```

Comment:
Again, thanks a lot for this PR. But before integrating it, I believe we need to at least come up with a way to make sure the newly added comment follows the proper indentation... example:


https://github.com/vscode-shellcheck/vscode-shellcheck/assets/29582865/dd909467-d552-4b16-a848-484ef3411121

Do you think you can handle this?

Reply:
I am using `shfmt` and did not notice that.

I fixed it to take indentation into comment line!